azure-pipelines.yml

```diff
@@ -0,0 +1,46 @@
+trigger:
+- master
+
+pr:
+  autoCancel: true
+  branches:
+    include:
+    - master
+
+jobs:
+- job: Xcode
+  displayName: 'Xcode build and tests'
+  pool:
+    vmImage: macOS-10.14
+    demands: xcode
+  steps:
+  - script: ./scripts/bluepill.sh test
+    displayName: 'Run BluePill tests'
+  - task: PublishTestResults@2
+    inputs:
+      testResultsFormat: 'JUnit'
+      testResultsFiles: 'build/reports/*.xml' 
+      mergeTestResults: false
+      failTaskOnFailedTests: false
+  - script: ./scripts/bluepill.sh build
+    displayName: 'Build BluePill'
+  - script: ls -alR
+    displayName: 'List stuff'
+  - task: CopyFiles@2
+    inputs:
+      contents: 'build/*.zip'
+      targetFolder: '$(Build.artifactStagingDirectory)'
+  - task: PublishBuildArtifacts@1
+    inputs:
+      pathtoPublish: '$(Build.ArtifactStagingDirectory)'
+
+- job: Bazel
+  displayName: 'Bazel build'
+  pool:
+    vmImage: macOS-10.14
+    demands: xcode
+  steps:
+  - script: 'brew install bazel'
```

Comment:
Do we want to pin bazel version here?

Reply:
Let's keep it unpinned for now - if we see Bazel breaking us we can pin then.